src/main/java/net/mcjukebox/plugin/bukkit/listeners/RegionListener.java

```diff
@@ -80,12 +80,14 @@ public void onMove(PlayerMoveEvent e){
                 String lastShow = utils.getURL(playerInRegion.get(e.getPlayer().getUniqueId()));
                 playerInRegion.remove(e.getPlayer().getUniqueId());
 
-                if(lastShow.toCharArray()[0] == '@') {
+                if (lastShow == null) {
```

Comment:
This should be checking if `lastShow == null  || lastShow.toCharArray()[0] != '@'`, since it really means lastSource. Maybe it should be renamed.